content/geometry/PolygonUnionArea.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * Author: black_horse2014, chilli
+ * Date: 2019-10-29
+ * License: CC0
+ * Source:
```

Comment:
Where did you get this from? Just to double-check that it's indeed fine with CC0.

Reply:
Hmm, accidentally left that in here. Unfortunately, I don't know where the original source is from. The implementation I took the most from is: https://codeforces.com/gym/101673/submission/50481926 (blackhorse_2014's submission on that problem).

Considering that many of the implementations (the short ones at least) used the same conceptual implementation that's very similar, I suspect there's a single original source. Unfortunately, it seems to be fairly difficult to find.